### PR TITLE
Document volume options, which are actually parsed and set

### PR DESCRIPTION
**Pull request by pablofsf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,12 +151,12 @@ config {
 }
 ```
 
-* **volumes** - (Optional) A list of host_path:container_path strings to bind host paths to container paths.
+* **volumes** - (Optional) A list of host\_path:container\_path:options strings to bind host paths to container paths. Named volumes are not supported.
```
Comment (jrasell) [inline README.md:154]:
```suggestion
* **volumes** - (Optional) A list of host_path:container_path:options strings to bind host paths to container paths. Named volumes are not supported.
```

Comment (pablofsf) [reply README.md]: Fixed and rebased! Thank you for looking into this one!

**Comment by pablofsf** [comment]
The failure has nothing to do with this PR. Fixed in one of the commits in https://github.com/hashicorp/nomad-driver-podman/pull/129

**Review by jrasell — APPROVED** [review]
LGTM, thanks @pablofsf!

**Pull request merged**